src/server/routes/index.js

```diff
@@ -1,7 +1,14 @@
 import Catchments from './catchments.js'
 import Licences from './licences.js'
+import Region from './regions.js'
 import Rivers from './rivers.js'
 import Static from './static.js'
 
-export const apiPrefixRoutes = [...Licences, ...Rivers, ...Catchments]
+export const apiPrefixRoutes = [
+  ...Licences,
+  ...Region,
+  ...Rivers,
+  ...Catchments
+]
```

Comment:
I worry about the performance of this - I think Array.concat would exhibit better execution speed. See https://jonlinnell.co.uk/articles/spread-operator-performance. Might not make too much difference, but worth being aware of